vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -1,3 +1,177 @@
 ---
-sidebarDepth: 1
+sidebarDepth: 2
 ---
+
+# Bundle Management - ent bundle CLI
+The Entando bundle CLI extends the functionality of **ent** with a modular bundle management system. Starting with Entando 7.1, ent bundle orchestrates the lifecycle of a project, whether it comes with a single component or many, and packages them into convenient recognizable bundles that can be inserted into any Entando Application. This modular approach takes advantage of a single project descriptor and repository, along with centralized [management of APIs](ent-api.md) and [services for DB and Keycloak](ent-svc.md).
+
+This document covers ent bundle operations and how it manages the lifecycle of a bundle, organized by the steps required.
+
+1. [Initialization](#initialization)
+2. [Build](#build)
+3. [Run](#run)
+4. [Package](#package)
+5. [Publish](#publish)
+6. [ECR Deploy](#ecr-bundle-commands)
+
+### Bundle Lifecycle Overview
+A single JSON descriptor file can define the parameters of an Entando project, but the life of an Entando Bundle begins at initialization. The ent init step sets up the structure and scaffolding needed for a new bundle. This tool allows a project to be intialized directly from the Entando Hub, speeding up the development process. The second step builds the project components for the micro frontends (MFE) and microservices (MS), whether you have a single component or many MSs and MFEs. They are built in parallel, using processes that are dependent on your stack, and given version numbers filtered by type and name. 
+
+The next step runs the components, processed in the same way by type and name, resulting in log files for each. The ent pack step generates the artifacts and builds the Docker images. A single Docker image is built for the bundle and all its microservices. Then Docker images are built for each micro frontend. Finally, these images are published to a Docker repository. 
```

Comment:
Sentence about Docker images is not correct. We build:
- one Docker image for the bundle (that contains also MFEs files)
- one Docker image for each microservice.